### ISSUE TITLE
Negative values in the raw outputs of ClassificationModel when tested on a devset. 

### ISSUE THREAD

**Issue by akrahdan** [ThilinaRajapakse/simpletransformers#1363]
**Describe the bug**
I am getting negative values in the raw outputs of ClassificationModel when tested on a devset. I am using the electra-base pretrained model for a multi-class classification task,  and haven't gotten any accuracy scores above 55%. 

If, as indicated in the shot below, the raw outputs contain so many negative values, shouldn't we be worried?
**To Reproduce**

**Expected behavior**

One will expect to see only positive values since they are the outputs from the CrossEntropyLoss function, as implemented in the predict function of the ClassificationModel.

**Screenshots**

<img width="631" alt="Screenshot 2022-02-13 at 2 29 28 PM" src="https://user-images.githubusercontent.com/6215902/153774002-dc003069-6ec7-4c69-8758-97473c904588.png">

**Desktop (please complete the following information):**
 - OS

**Additional context**
